content/es/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -0,0 +1,348 @@
+   EOF
+   ```
+
+{{< note >}}
+Para actualizar kubectl a otra versión, será necesario modificar la versión en `/etc/yum.repos.d/kubernetes.repo`
+antes de ejecutar `yum update`. Este procedimiento se describe con más detalle en [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
+{{< /note >}}
+
+2. Instalar kubectl utilizando `yum`:
+
+   ```bash
+   sudo yum install -y kubectl
+   ```
+
+{{% /tab %}}
+
+{{% tab name="SUSE-based distributions" %}}
+
+1. Agregar Kubernetes al repositorio `zypper`. Si deseas usar una versión de Kubernetes
+   diferente a {{< param "version" >}}, reemplaza {{< param "version" >}} con
+   la versión deseada en el siguiente comando:
+
+   ```bash
+   # Lo siguiente reemplaza cualquier configuración existente en /etc/zypp/repos.d/kubernetes.repo
+   cat <<EOF | sudo tee /etc/zypp/repos.d/kubernetes.repo
+   [kubernetes]
+   name=Kubernetes
+   baseurl=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/
+   enabled=1
+   gpgcheck=1
+   gpgkey=https://pkgs.k8s.io/core:/stable:/{{< param "version" >}}/rpm/repodata/repomd.xml.key
+   EOF
+   ```
+
+{{< note >}}
+Para actualizar kubectl a otra versión será necesario modificar la versión en `/etc/zypp/repos.d/kubernetes.repo`
+antes de ejecutar `zypper update`. Este procedimiento se describe con más detalle en [Changing The Kubernetes Package Repository](/docs/tasks/administer-cluster/kubeadm/change-package-repository/).
+{{< /note >}}
+
+   2. Instalar kubectl usando `zypper`:
+
+      ```bash
+      sudo zypper install -y kubectl
+      ```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+### Instalación usando otro administrador de paquetes
+
+{{< tabs name="other_kubectl_install" >}}
+{{% tab name="Snap" %}}
+Si utilizas Ubuntu o alguna distribución que soporte el administrador de
+páquetes [snap](https://snapcraft.io/docs/core/install), kubectl
+está disponible como una aplicación de [snap](https://snapcraft.io/).
+
+```shell
+snap install kubectl --classic
+kubectl version --client
+```
+
+{{% /tab %}}
+
+{{% tab name="Homebrew" %}}
+Si utilizas [Homebrew](https://docs.brew.sh/Homebrew-on-Linux) en Linux,
+kubectl está disponible para su [instalación](https://docs.brew.sh/Homebrew-on-Linux#install).
+
+```shell
+brew install kubectl
+kubectl version --client
+```
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+## Verificar la configuración de kubectl
+
+{{< include "included/verify-kubectl.md" >}}
+
+## Configuraciones opcionales y plugins de kubectl
+
+### Habilitar el autocompletado en la shell
+
+Kubectl tiene soporte para autocompletar en Bash, Zsh, Fish y Powershell,
+lo que puede agilizar el tipeo.
+
+A continuación están los procedimientos para configurarlo en Bash, Fish y Zsh.
+
+{{< tabs name="kubectl_autocompletion" >}}
+{{< tab name="Bash" include="included/optional-kubectl-configs-bash-linux.md" />}}
+<!-- TODO: included/optional-kubectl-configs-fish.md -->
```

Comment:
/hold
until we merge the PR to address this translation related: https://github.com/kubernetes/website/pull/44382